client/src/mvc/library/library-libraryrow-view.js

```diff
@@ -243,7 +243,7 @@ var LibraryRowView = Backbone.View.extend({
                         <td style="color:grey;"><%- library.get("name") %></td>
                     <% } else { %>
                         <td>
-                            <a href="folders/<%- library.get("root_folder_id") %>"><%- library.get("name") %></a>
+                            <a href="/library/folders/<%- library.get("root_folder_id") %>"><%- library.get("name") %></a>
```

Comment:
This should actually probably be a `getAppRoot()`-based URL?  I have not tried it, but does this break when serving at a proxy?  e.g. `http://myinfrastructureurl.org/galaxy/library/folders/`?

Reply:
> This should actually probably be a `getAppRoot()`-based URL? I have not tried it, but does this break when serving at a proxy? e.g. `http://myinfrastructureurl.org/galaxy/library/folders/`?

yes, that's right. Should be better now